### PR TITLE
ACRS-25 Set ACRS service in hof-rds-api

### PR DESCRIPTION
**Pull request by TemitopeAyokuHO**
## What? 
[ACRS-25](https://collaboration.homeoffice.gov.uk/jira/browse/ACRS-25) Create database for acrs app

## Why? 
To save applications into database 

## How? 
- Created Database table for saved application in postgres fo ACRS app
- Added a migration file to take snaps shot of table in postgres
- Added a seed file to populate the table once the app start
- Added the pr template file.

## Anything Else? (optional)

##Check list
- [x] I have reviewed my own pull request for linting issues (e.g. adding new lines)
- [ ] I have written tests (if relevant) 
     note:   i will write a new test to cover all test cases
- [x] I have created a JIRA number for my branch
- [x] I have created a JIRA number for my commit
- [x] I have followed the chris beams method for my commit https://cbea.ms/git-commit/
here is an [example commit](https://github.com/UKHomeOfficeForms/hof/commit/810959f391187c7c4af6db262bcd143b50093a6e)
- [x] Ensure drone builds are green especially tests
- [x] I will squash the commits before merging

### CODE REVIEW

**Review by Rhodine-orleans-lindsay — CHANGES_REQUESTED** [review]
See comments. Could you also add the new pull_request_template.md file to the repo and could you add the checklist section to your own PR description (it helps the reviewer know what steps you have covered)?

File: services/acrs/seeds/01_saved_applications.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          email: 'test.dev@digital.homeoffice.gov.uk',
+          uan: '',
+          brp: 'RJ0000000',
+          date_of_birth: '2000/01/01',
+          session: '{}',
+          submitted_at: '2023/09/09'
+        },
+        {
+          email: 'test.dev@digital.homeoffice.gov.uk',
+          uan: '',
```
[the remark this answers is not in the document]
Comment (TemitopeAyokuHO) [reply services/acrs/seeds/01_saved_applications.js]: Changed to null

File: services/acrs/seeds/01_saved_applications.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  // Deletes ALL existing entries
+  return knex('saved_applications').del()
+    // eslint-disable-next-line func-names
+    .then(function () {
+      // Inserts seed entries
+      return knex('saved_applications').insert([
+        {
+          email: 'test.dev@digital.homeoffice.gov.uk',
+          uan: '0000-0000-0000-0000',
+          brp: '',
```
[the remark this answers is not in the document]
Comment (TemitopeAyokuHO) [reply services/acrs/seeds/01_saved_applications.js]: Changed to null

File: services/acrs/migrations/20240419213237_acrs.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+exports.up = function(knex) {
+    return knex.schema.createTable('saved_applications', table => {
+        table.increments();
+        table.string('brp');
+        table.string('uan');
```
[the remark this answers is not in the document]
Comment (TemitopeAyokuHO) [reply services/acrs/migrations/20240419213237_acrs.js]: Changed to default null

File: pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+
```
Comment (Rhodine-orleans-lindsay) [inline pull_request_template.md:1]: remove empty line

Comment (TemitopeAyokuHO) [reply pull_request_template.md]: done

**Review by Rhodine-orleans-lindsay — APPROVED** [review]

**Pull request merged**